client/signup/steps/jpo-summary/index.jsx

```diff
@@ -0,0 +1,147 @@
+			genre: payload[1].jpoSiteType.genre,
+			businessName: payload[1].jpoSiteType.addressInfo.businessName,
+			businessAddress: payload[1].jpoSiteType.addressInfo.streetAddress,
+			businessCity: payload[1].jpoSiteType.addressInfo.city,
+			businessState: payload[1].jpoSiteType.addressInfo.state,
+			businessZipCode: payload[1].jpoSiteType.addressInfo.zipCode,
+			homepageFormat: payload[2].jpoHomepage,
+			addContactForm: payload[3].jpoContactForm
+		};
+
+		return formattedPayload;
+	},
+
+	completeOnboarding() {
+		console.log( this.getFormattedPayload() );
+	},
+
+	goToStepOne() {
+		this.props.goToStep( 1 );
+	},
+
+	goToStepTwo() {
+		this.props.goToStep( 2 );
+	},
+
+	goToStepThree() {
+		this.props.goToStep( 3 );
+	},
+
+	goToStepFour() {
+		this.props.goToStep( 4 );
+	},
+
+	skipStep() {
+		this.props.goToNextStep();
+	},
+
+	renderStepContent() {
+		const connectionToJetpackComplete = true;
+
+		const checkMark = <Gridicon
+			className="plan-features__item-checkmark"
+			size={ 18 } icon="checkmark" />;
+
+		return (
+			<div className="jpo__summary-wrapper">
+				<table>
+					<tbody>
+						<tr>
+							<td>
+								<div className="jpo__summary-col-header">Steps you've completed:</div>
+								<ul>
+									{ 'undefined' !== typeof this.props.signupProgress[0].jpoSiteTitle ? <li>{ checkMark } { translate( 'Site Title & Description' ) }</li> : null }
+									{ 'undefined' !== typeof this.props.signupProgress[1].jpoSiteType ? <li>{ checkMark } { translate( 'Type of Site' ) }</li> : null }
+									{ 'undefined' !== typeof this.props.signupProgress[2].jpoHomepage ? <li>{ checkMark } { translate( 'Type of Homepage' ) }</li> : null }
+									{ 'undefined' !== typeof this.props.signupProgress[3].jpoContactForm ? <li>{ checkMark } { translate( 'Contact Us Form' ) }</li> : null }
+									{ connectionToJetpackComplete ? <li>{ checkMark } { translate( 'Connection to Jetpack' ) }</li> : null }
+								</ul>
+							</td>
+							<td>
+								<div className="jpo__summary-col-header">Configure more of your site:</div>
+								<ul>
+									{ 'undefined' === typeof this.props.signupProgress[0].jpoSiteTitle ? <li onClick={ this.goToStepOne }>{ translate( 'Site Title & Description' ) }</li> : null }
+									{ 'undefined' === typeof this.props.signupProgress[1].jpoSiteType ? <li onClick={ this.goToStepTwo }>{ translate( 'Type of Site' ) }</li> : null }
+									{ 'undefined' === typeof this.props.signupProgress[2].jpoHomepage ? <li onClick={ this.goToStepThree }>{ translate( 'Type of Homepage' ) }</li> : null }
+									{ 'undefined' === typeof this.props.signupProgress[3].jpoContactForm ? <li onClick={ this.goToStepFour }>{ translate( 'Contact Us Form' ) }</li> : null }
+									{ ! connectionToJetpackComplete ? <li>{ translate( 'Connection to Jetpack' ) }</li> : null }
+									<li>{ translate( 'Choose a Theme' ) }</li>
+									<li>{ translate( 'Add a Site Address' ) }</li>
```

Comment:
Hi! I've found a possible matching string that has already been translated **1** time:
`translate( 'Site Address', 'NULL', { context: 'NULL'} )` *ES Score: 9*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/jpo-flow-steps)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).